aesd-char-driver/Makefile

```diff
@@ -12,9 +12,11 @@ endif
 EXTRA_CFLAGS += $(DEBFLAGS)
 
 ifneq ($(KERNELRELEASE),)
-# call from kernel build system
+# call from kernel build system. If KERNELRELEASE is defined, we've been invoked from the kernel build system and can use its language.
+# Credit: Linux Device Drivers Chapter 2: Building and Running Kernels
 obj-m	:= aesdchar.o
 aesdchar-y := aesd-circular-buffer.o main.o
+KBUILD_OPTIONS += NO_BTF=1
```

Comment:
@sota6640 what was the reason for this change?

Reply:
I was getting an error regarding BTF generation due to vmlinux not being available. I followed this approach. https://askubuntu.com/questions/1348250/skipping-btf-generation-xxx-due-to-unavailability-of-vmlinux-on-ubuntu-21-04